.github/workflows/release.yml

```diff
@@ -53,23 +55,33 @@ jobs:
                   fi
                   echo "resolved=$RELEASE_VERSION" >> "$GITHUB_OUTPUT"
 
-            - name: Bump versions and commit
+            - name: Resolve version
               env:
                   INPUT_VERSION: ${{ steps.resolve_version.outputs.resolved }}
               run: |
-                  git config user.name "github-actions[bot]"
-                  git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
                   node ./scripts/release/version.js "$INPUT_VERSION"
                   RESOLVED_VERSION=$(jq -r .version package.json)
-                  git commit -am "chore: release v$RESOLVED_VERSION"
-                  git push origin HEAD
+                  echo "RESOLVED_VERSION=$RESOLVED_VERSION" >> "$GITHUB_ENV"
+
+            - name: Set git identity (Actions bot)
+              run: |
+                  git config user.name "github-actions[bot]"
+                  git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Should that email be a secret as well?

Reply:
I don't think so, this is generic github actions bot email